client/elements/text/sc-image.html

```diff
@@ -0,0 +1,113 @@
+<link rel="import" href="/bower_components/polymer/polymer.html">
+<link rel="import" href="/bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" href="/bower_components/iron-meta/iron-meta.html">
+
+<link href="/node_modules/viewerjs/dist/viewer.min.css" rel="stylesheet">
+<link rel="import" href="/elements/text/sc-viewer-import.html">
+
+<dom-module id="sc-image">
+  <template>
+
+    <style>
+    </style>
+    
+    <iron-meta id="meta"></iron-meta>
+
+    <iron-ajax
+        id="ajax"
+        debounce-duration="500"
+        handle-as="json"
+        loading="{{isLoading}}"
+        last-error="{{lastError}}"
+        last-response="{{imagesData}}"
+        on-response="_gotData"></iron-ajax>
```

Comment:
I'd like to standardize the names of these response handlers. Currently we have a few (`_didRespond`, `_handleResponse` `_didReceiveResponse`, but the most common one is `_didRespond`. Take your pick, but I wouldn't introduce another one.